tests/Report/JunitReporterTest.php

```diff
@@ -42,47 +38,31 @@
 
     protected function setUp()
     {
-        $this->reporter = new JunitReporter();
+        parent::setUp();
+
         $this->xsd = file_get_contents(__DIR__.'/../../doc/junit-10.xsd');
```

Comment:
This is used in `assertFormat()` only. I would replace the property (and the need to override `setUp()`) with a simple variable in `assertFormat()`.

Reply:
actually, this data is constant across tests, no need to read it from IO each time per case

Reply:
moved to static